microsoft_entra_id/README.md

```diff
@@ -75,7 +73,7 @@ You can view these events in [Event Explorer][5].
 
 Need help? Contact [Datadog support][4].
 
-[1]: https://docs.microsoft.com/en-us/azure/active-directory/reports-monitoring/overview-reports#activity-reports
+[1]: https://docs.microsoft.com/azure/active-directory/reports-monitoring/overview-reports#activity-reports
```

Comment:
Should this be https://learn.microsoft.com/entra/identity/monitoring-health/overview-monitoring-health#activity-reports ?